### PR TITLE
Show ACH processor

### PR DESCRIPTION
**Pull request by YodaLightsabr**
My attempt at #3210

### CODE REVIEW

File: app/controllers/admin_controller.rb [hunk 1]
```diff
@@ -522,13 +522,14 @@ def ach_approve
...
+      confirmation_number: params[:confirmation_number],
+      processor: current_user
     }
     ach_transfer = AchTransferService::Approve.new(attrs).run
 
     redirect_to ach_start_approval_admin_path(ach_transfer), flash: { success: "Success" }
-  rescue => e
-    redirect_to ach_start_approval_admin_path(params[:id]), flash: { error: e.message }
+  # rescue => e
+  #   redirect_to ach_start_approval_admin_path(params[:id]), flash: { error: e.message }
```
Comment (cjdenio) [inline app/controllers/admin_controller.rb]:
```suggestion
  rescue => e
    redirect_to ach_start_approval_admin_path(params[:id]), flash: { error: e.message }
```

**Review by cjdenio — APPROVED** [review]

**Review by garyhtou — APPROVED** [review]
Looks good!

**Comment by cjdenio** [comment]
oop @YodaLightsabr could you rename the field to `processed_by` instead of `processor`? That way it's more consistent with other models, and more evident it's referring to a person

**Comment by YodaLightsabr** [comment]
How could I go about doing it now that it's been merged? Would it mess up any ACH transfers made in the last few days?

**Comment by cjdenio** [comment]
There's only been 1 ACH transfer since— I think the easiest solution is to revert this PR and roll back the database schema

**Pull request merged**